.gitignore

```diff
@@ -67,6 +67,7 @@ coverage.all
 /integrations/mssql.ini
 /node_modules
 /modules/indexer/issues/indexers
+/modules/indexer/code/bleve.index/
```

Comment:
Hmm... rather than add this to git ignore we should fix whatever is making this to not do so or to clean up after itself...

Reply:
`make test-stqlite` generate it. Already talked with lunny about this

Reply:
it's made by make test.

See https://github.com/go-gitea/gitea/pull/9515

To be completely clear about this, IMHO no test should ever write to the source code tree.

Reply:
```suggestion
```